lib/BrowserFetcher.js

```diff
@@ -137,14 +137,32 @@ class BrowserFetcher {
     if (!(await existsAsync(this._downloadsFolder)))
       await mkdirAsync(this._downloadsFolder);
     try {
-      await downloadFile(url, zipPath, progressCallback);
-      await extractZip(zipPath, folderPath);
+      if (os.arch() === 'arm64'){
+        fs.stat('/usr/bin/chromium-browser', function(err, stats) {
+          if (stats === undefined){
+            console.error('Chromium Binary is not available for aarch64, Download it manually.');
+            const result = require('child_process').execSync('cat /etc/lsb-release | grep -in "DISTRIB_ID" | cut -f 2 -s -d =').toString().trim();
```

Comment:
/etc/lsb-release is not part of my Ubuntu default OS. I've looked around and it isn't obvious how you would reliability test for the OS distro name in a standard way. It may be simpler to leave out this check, and just have the console.error lines below:

"If you are on Ubuntu, you can install with:"...

Reply:
@rhenwood-arm As per your suggestion, I have modified the title and removed the check for ubuntu.